verticles/graphql/src/main/java/com/gentics/mesh/graphql/type/field/FieldDefinitionProvider.java

```diff
@@ -302,14 +303,14 @@ public GraphQLFieldDefinition createListDef(GraphQLContext context, ListFieldSch
 				Map<String, ?> filterArgument = env.getArgument("filter");
 				Stream<NodeContent> nodes = nodeList.getList().stream().map(item -> {
 					Node node = item.getNode();
-					List<String> languageTags = Collections.emptyList();
+					List<String> languageTags;
 					if (container instanceof NodeGraphFieldContainer) {
 						languageTags = Arrays.asList(container.getLanguage().getLanguageTag());
+					} else if (container instanceof Micronode) {
+						Micronode micronode = (Micronode)container;
+						languageTags = Arrays.asList(micronode.getContainer().getLanguage().getLanguageTag());
 					} else {
-						// Other containers (e.g. micronodes do not have a language thus we can't use that language to define the loaded language variant. We
-						// thus fallback to the default mesh language.
-						String defaultLanguage = Mesh.mesh().getOptions().getDefaultLanguage();
-						languageTags = Arrays.asList(defaultLanguage);
+						throw new RuntimeException("container can only be NodeGraphFieldContainer or Micronode");
```

Comment:
Throw GenericRestExceptions. Other Exceptions might be hard to convert to errors in the error field of the graphql response.